app/helpers/application_helper.rb

```diff
@@ -153,36 +153,44 @@ def available_courses
       {
         badge_image_url: 'badge-html-css.svg',
         badge_alt_text: 'html css badge',
-        title: 'HTML + CSS'
+        title: 'HTML + CSS',
+        path: '/courses/html-and-css'
       },
       {
         badge_image_url: 'badge-javascript.svg',
         badge_alt_text: 'javascript badge',
-        title: 'JavaScript'
+        title: 'JavaScript',
+        path: '/courses/javascript'
       },
       { badge_image_url: 'badge-git.svg',
         badge_alt_text: 'git badge',
-        title: 'Git'
+        title: 'Git',
+        path: '/courses/web-development-101#git-basics'
       },
       { badge_image_url: 'badge-database.svg',
         badge_alt_text: 'databases badge',
-        title: 'Databases'
+        title: 'Databases',
+        path: '/courses/databases'
       },
       { badge_image_url: 'badge-ruby.svg',
         badge_alt_text: 'ruby badge',
-        title: 'Ruby'
+        title: 'Ruby',
+        path: '/courses/ruby-progrmming'
```

Comment:
Sorry, just noticing this. You have a typo here, should be `/ruby-programming` not `/ruby-progrmming`